src/components/PoolCard/index.tsx

```
@@ -585,536 +388,162 @@ const SwapButtonLabel = styled.div`
-  // if (
-  //   !info ||
-  //   info.poolBals.reduce((acc: bigint, val: bigint) => acc + val, BigInt(0)) ===
-  //     BigInt(0)
-  // )
-  //   return null;
+  const tokAIcon = isWVOIf(tokA) ? (
+    <PairIconImage
+      src={`https://asset-verification.nautilus.sh/icons/0.png`}
+      alt={`VOI icon`}
+    />
+  ) : !!tokA?.verified ? (
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)